src/device.rs

```diff
@@ -46,7 +46,7 @@ impl<C> Flow for DeviceFlow<C> {
     }
 }
 impl<C> DeviceFlow<C>
-    where C: BorrowMut<hyper::Client>
+    where C: BorrowMut<hyper::Client<hyper::client::HttpConnector, hyper::Body>>
```

Comment:
I think it would be nice to alias this type. Maybe ```type OauthHyperClient = hyper::Client<hyper::client::HttpConnector, hyper::Body>```.

Also this needs to be moved to the Http**s** type.